packages/notificator/notificator.go

```diff
@@ -121,6 +129,22 @@ func SendNotifications() {
 	}
 }
 
+func sendUserStats(user int64, stats []notificationRecord) {
+	rawStats, err := json.Marshal(stats)
+	if err != nil {
+		log.WithFields(log.Fields{"type": consts.MarshallingError, "error": err}).Error("notification statistic")
+	}
+
+	ok, err := publisher.Write(user, string(rawStats))
+	if err != nil {
```

Comment:
if err != nil || !ok {
    log.WithFields(log.Fields{"type": consts.CentrifugoError,}).Error("writing to centrifugo")